src/main/java/com/fontys/api/entities/Team.java

```diff
@@ -1,28 +1,35 @@
 package com.fontys.api.entities;
 
+import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
+import javax.persistence.*;
```

Comment:
I think it's beter to have specific endpoint instead of importing the whole package at once